modules/ROOT/pages/maven-preferences-reference.adoc

```diff
@@ -3,16 +3,26 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio allows you to customize your preferences for working with Maven.
+In Studio, you can personalize your preferences when working with Maven. Studio reads the `settings.xml` file, which details the configurations Maven uses during its operation. The `settings.xml` file enables you to tailor the behavior of the build process for all the projects on your system. With the `settings.xml` file, you can configure:
 
-image::maven-preferences-reference.png["*Maven Preferences Reference* window accessed from the *Settings...* option, from the *Anypoint Studio* menu."]
+* Dependency and plugin repositories
+* Proxy servers
+* Authentication
+* Plugins
+
+See https://maven.apache.org/settings.html[Maven setting references^] for additional information. 
+
+To configure Maven setting preferences, go to *Anypoint Studio > Settings…* and click *Maven*.
```

Comment:
I would change this instruction... this is Mac specific directions.

Reply:
Thanks for the clarification!